Radius(x).py

```diff
@@ -41,8 +78,13 @@
 axes.grid()
 axes.set_xlabel("Axial coordinate x (m)", fontsize=16)
 axes.set_ylabel("Radius (m)", fontsize=16)
-axes.set_title("Radius of the jet as a function of the axial coordinate ", fontsize=16)
-#axes.set_title("CANDY-V001 / PLA", fontsize=16)
+
+#Choose one title
+#axes.set_title("Radius of the jet as a function of the axial coordinate ", fontsize=16)
+#axes.set_title("Super Floss Maxx  / PP ", fontsize=16)
+#axes.set_title("Super Floss Maxx  / PLA", fontsize=16, y=1.)
+#axes.set_title("CANDY-V001  / PP", fontsize=16, y=1.)
+axes.set_title("CANDY-V001  / PLA", fontsize=16, y=1.)
```

Comment:
Once the machine and polymer properties will be loaded from files, we could have here an automatic generation of the title to display machine name and polymer type.
We will have to design the files accordingly to have the information we need.